docs/installation.rst

```diff
@@ -0,0 +1,16 @@
+Installation
+============
+
+If you are a researcher that is interested in running analyses on the REANA
+cloud, all you need to install is the ``reana-client``:
+
+.. code-block:: console
+
+   $ pip install reana-client
```

Comment:
Since we don't have a PyPI release yet, maybe add pip install line from github for now.